### PR TITLE
Added limits for resource manager/resource groups

### PR DESCRIPTION
**Pull request by rjmax**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@rjmax__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ryjones). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: articles/azure-subscription-service-limits.md [hunk 1]
```diff
@@ -165,6 +174,38 @@ This document specifies some of the most common Microsoft Azure limits. Note tha
 
 <sup>2</sup>Input endpoints are used to allow communication to the virtual machines that is external to the containing cloud service. Virtual machines within the same cloud service automatically allow communication between all UDP and TCP ports for internal communication.
 
+## Resource Group Limits
+
+<table cellspacing="0" border="1">
+<tr>
+   <th align="left" valign="middle">Resource</th>
```
Comment (IlyaGrebnov) [inline articles/azure-subscription-service-limits.md]: Information is outdated. It is 800 actually :)

Comment (rjmax) [inline articles/azure-subscription-service-limits.md]: Fixed :)

File: articles/azure-subscription-service-limits.md [hunk 2]
```diff
@@ -97,7 +97,16 @@ This document specifies some of the most common Microsoft Azure limits. Note tha
    <td valign="middle"><p>500</p></td>
    <td valign="middle"><p>500</p></td>
 </tr>
-
+<tr>
+   <td valign="middle"><p>Resource Manager API Reads</p></td>
+   <td valign="middle"><p>3200 per hour</p></td>
```
Comment (yangl900) [inline articles/azure-subscription-service-limits.md]: typo. this should be 32000 per hour.

**Pull request merged**